core/actor/src/main/scala/net/liftweb/actor/LAFuture.scala

```diff
@@ -171,7 +171,7 @@ class LAFuture[T](val scheduler: LAScheduler = LAScheduler, context: Box[LAFutur
   /**
    * Has the future been aborted
    */
-  def aborted_? = synchronized {satisfied}
+  def aborted_? = synchronized {aborted}
```

Comment:
Oh my. This.... is a pretty bad bug. 🤔 

@lift/committers Thoughts on incorporating this fix back to 3.0 since this certainly isn't the _intended_ behavior?

Reply:
Oh wait, we... we never actually flip `aborted` to true as far as I can see, either. It looks like we should do that in `abort()` yeah?

Reply:
We do, in the `fail` method, which `abort` also falls.

Reply:
So, re: backporting this, the problem is it really changes a fundamental behavior… By literally flipping the previous behavior <_<

But, the existing behavior is *so wrong* that it might still be worth it? This is a tough call.

Reply:
Yeah I'm over here watching and eating popcorn cuz I ain't got a clue.

Reply:
Backports are generally

Reply:
Per our support doc, ‘API-incompatible changes will only ship as a part of a major release’… This… is pretty API-incompatible…

So perhaps the big question isn't “do we backport this to Lift 3.0” but… Do we merge it into Lift 3.3 at all?

My feeling is yes to both… But it's a pretty fuzzy feeling.

Reply:
I'm leaning toward "merging for Lift 3.3, not backporting, and including BIG BOLD TEXT in the release notes for 3.3" - mostly because this never worked as intended. I'd also love a test to make sure we don't do something foolish like this in the future. :(

Reply:
+1 not backporting it

Reply:
:thumbsup: I'm down with that.

Re: something foolish like this, thus code review :)